packages/auth/src/accessToken/service.test.ts

```diff
@@ -207,21 +207,21 @@ describe('Access Token Service', (): void => {
       await token.$query(trx).patch({ expiresIn: 1000000 })
       const result = await accessTokenService.revoke(token.managementId)
       expect(result).toBeUndefined()
-      token = await AccessToken.query(trx).findById(token.id)
+      token = (await AccessToken.query(trx).findById(token.id)) as AccessToken
```

Comment:
Those casts seem unnecessary if token is expected to be undefined.
It also makes it seem like the query/findById api could use enhancement.

Reply:
You are right. I was sloppy here. I just added the type cast for the compiler to be happy but I should have looked at the entire code block and improved it.

Reply:
Moaning compilers!

I was just poking around your draft after looking to see if MY pr was merged: 
https://github.com/interledger/rafiki/pull/575